examples/complete/variables.tf

```diff
@@ -1,8 +1,3 @@
-variable "region" {
-  type        = string
-  description = "AWS Region"
-}
```

Comment:
How come this is removed ?

Reply:
Oh I see, it's unused.

Reply:
It's not used anywhere (we don't use the AWS provider)